src/api/java/appeng/api/util/AECableType.java

```diff
@@ -29,32 +29,32 @@
 	/**
 	 * No Cable present.
 	 */
-	NONE,
+	NONE( AECableVariant.NONE, AECableSize.NONE ),
 
 	/**
 	 * Connections to this block should render as glass.
 	 */
-	GLASS,
+	GLASS( AECableVariant.GLASS, AECableSize.THIN ),
 
 	/**
 	 * Connections to this block should render as covered.
 	 */
-	COVERED,
+	COVERED( AECableVariant.COVERED, AECableSize.NORMAL ),
 
 	/**
 	 * Connections to this block should render as smart.
 	 */
-	SMART,
+	SMART( AECableVariant.SMART, AECableSize.NORMAL ),
 
 	/**
 	 * Smart Dense Cable, represents a tier 2 block that can carry 32 channels.
 	 */
-	DENSE_COVERED,
+	DENSE_COVERED( AECableVariant.COVERED, AECableSize.DENSE ),
 
 	/**
 	 * Smart Dense Cable, represents a tier 2 block that can carry 32 channels.
 	 */
-	DENSE_SMART;
+	DENSE_SMART( AECableVariant.SMART, AECableSize.DENSE );
 
 	public static final AECableType[] VALIDCABLES = {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2386)